tests/tck/features/match/Path.feature

```diff
@@ -38,10 +38,296 @@ Feature: Matching paths
       | count(p) | count(p2) |
       | 966      | 966       |
 
+  Scenario: overlapping aliases variables
+    When executing query:
+      """
+      match p = (v1:Label_3)-[e1]-(v1)
+      return count(p)
+      """
+    Then the result should be, in any order:
+      | count(p) |
+      | 28       |
+    When executing query:
+      """
+      match p2 = (v2:Label_5)-[e2]-(v2)
+      return count(p2)
+      """
+    Then the result should be, in any order:
+      | count(p2) |
+      | 43        |
```

Comment:
Any difference between these two test cases?

Reply:
These count results are used to prove the correctness of the following query.